### PR TITLE
docs: translate isValidElement

### LINKED ISSUE #552: `isValidElement`
Page: https://id.react.dev/reference/react/isValidElement

### PR DESCRIPTION
**Pull request by renomureza**
Closes #552

## Description

Translate the isValidElement page.
Page URL: https://id.react.dev/reference/react/isValidElement

### CODE REVIEW

File: src/content/reference/react/isValidElement.md [hunk 1]
```diff
@@ -16,72 +16,72 @@ const isElement = isValidElement(value)
...
+[Lihat lebih banyak contoh di bawah ini.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `value`: The `value` you want to check. It can be any a value of any type.
+* `value`: Sebuah nilai yang ingin diperiksa. Itu dapat berupa nilai apa pun dari jenis apa pun.
 
-#### Returns {/*returns*/}
+#### Pengembalian {/*returns*/}
```
Comment (resir014) [inline src/content/reference/react/isValidElement.md:44]:
```suggestion
#### Kembalian {/*returns*/}
```

Use "kembalian", not "pengembalian."

https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320

File: src/content/reference/react/isValidElement.md [hunk 2]
```diff
@@ -16,72 +16,72 @@ const isElement = isValidElement(value)
...
 
-`isValidElement` returns `true` if the `value` is a React element. Otherwise, it returns `false`.
+`isValidElement` mengembalikan `true` jika `value` adalah elemen React. Jika bukan, ia mengembalikan `false`.
 
-#### Caveats {/*caveats*/}
+#### Peringatan {/*caveats*/}
```
Comment (resir014) [inline src/content/reference/react/isValidElement.md:48]:
```suggestion
#### Catatan penting {/*caveats*/}
```

Use "Catatan penting" instead. https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5768126

File: src/content/reference/react/isValidElement.md [hunk 3]
```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
...
-It is very uncommon to need `isValidElement`. It's mostly useful if you're calling another API that *only* accepts elements (like [`cloneElement`](/reference/react/cloneElement) does) and you want to avoid an error when your argument is not a React element.
+`isValidElement` sangat jarang diperlukan. Ini sangat berguna jika Anda memanggil API lain yang *hanya* menerima elemen (seperti halnya [`cloneElement`](/reference/react/cloneElement)) dan Anda ingin menghindari kesalahan saat argumen Anda bukan elemen React.
 
-Unless you have some very specific reason to add an `isValidElement` check, you probably don't need it.
+Kecuali Anda memiliki alasan yang sangat spesifik untuk menambahkan pemeriksaan `isValidElement`, Anda mungkin tidak membutuhkannya.
 
 <DeepDive>
 
-#### React elements vs React nodes {/*react-elements-vs-react-nodes*/}
+#### Element React vs *node* React {/*react-elements-vs-react-nodes*/}
```
Comment (resir014) [inline src/content/reference/react/isValidElement.md:99]:
```suggestion
#### Elemen React vs *node* React {/*react-elements-vs-react-nodes*/}
```

element -> elemen

**Review by resir014 — CHANGES_REQUESTED** [review]
Some requested changes. @renomureza

**Comment by renomureza** [comment]
> Some requested changes. @renomureza

Hi @resir014, i have fixed it.

File: src/content/reference/react/isValidElement.md [hunk 4]
```diff
@@ -16,72 +16,72 @@ const isElement = isValidElement(value)
...
 console.log(isValidElement(<MyComponent />)); // true
 
-// ✅ Values returned by createElement are React elements
+// ✅ Nilai yang dikembalikan oleh createElement adalah elemen React
 console.log(isValidElement(createElement('p'))); // true
 console.log(isValidElement(createElement(MyComponent))); // true
 ```
 
-Any other values, such as strings, numbers, or arbitrary objects and arrays, are not React elements.
+Nilai lainnya, seperti string, angka, atau objek arbitrer dan *array*, bukan elemen React.
```
Comment (mazipan) [inline src/content/reference/react/isValidElement.md:79]: Arbitrary ini apa ya @mhaidarhanif @resir014 @r17x ?

Comment (resir014) [reply src/content/reference/react/isValidElement.md:79]:
```suggestion
Nilai lainnya, seperti string, angka, atau objek sembarang dan *array*, bukan elemen React.
```

Per the above thread. Please change this to "sembarang".

**Review by resir014 — CHANGES_REQUESTED** [review]
One minor change. @renomureza

File: src/content/reference/react/isValidElement.md [hunk 5]
```diff
@@ -16,72 +16,72 @@ const isElement = isValidElement(value)
...
 
-Any other values, such as strings, numbers, or arbitrary objects and arrays, are not React elements.
+Nilai lainnya, seperti string, angka, atau objek sembarang dan *array*, bukan elemen React.
 
-For them, `isValidElement` returns `false`:
+Untuk nilai-nilai tersebut, `isValidElement` mengembalikan `false`: 
 
 ```js
-// ❌ These are *not* React elements
+// ❌ Ini adalah *bukan* elemen React
```
Comment (mhaidarhanif) [inline src/content/reference/react/isValidElement.md:84]: Minor changes

```suggestion
// ❌ Berikut ini *bukan* elemen React
```

File: src/content/reference/react/isValidElement.md [hunk 6]
```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
...
-#### React elements vs React nodes {/*react-elements-vs-react-nodes*/}
+#### Elemen React vs *node* React {/*react-elements-vs-react-nodes*/}
 
-When you write a component, you can return any kind of *React node* from it:
+Saat Anda menulis sebuah komponen, Anda dapat mengembalikan *node React* apa pun darinya:
 
 ```js
 function MyComponent() {
-  // ... you can return any React node ...
+  // ... anda dapat mengembalikan node React apapun ...
```
Comment (mhaidarhanif) [inline src/content/reference/react/isValidElement.md:105]: Minor change

```suggestion
  // ... Anda dapat mengembalikan node React apapun ...
```

File: src/content/reference/react/isValidElement.md [hunk 7]
```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
...
 }
 ```
 
-A React node can be:
+Node React dapat berupa:
```
Comment (mhaidarhanif) [inline src/content/reference/react/isValidElement.md:109]:
```suggestion
*Node* React dapat berupa:
```

File: src/content/reference/react/isValidElement.md [hunk 8]
```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
...
-- A React element created like `<div />` or `createElement('div')`
-- A portal created with [`createPortal`](/reference/react-dom/createPortal)
-- A string
-- A number
-- `true`, `false`, `null`, or `undefined` (which are not displayed)
-- An array of other React nodes
+- Elemen React yang dibuat seperti `<div />` atau `createElement('div')`
+- Portal yang dibuat dengan [`createPortal`](/reference/react-dom/createPortal)
+- Sebuah *string*
+- Sebuah Angka
```
Comment (mhaidarhanif) [inline src/content/reference/react/isValidElement.md:114]:
```suggestion
- Sebuah angka
```

File: src/content/reference/react/isValidElement.md [hunk 9]
```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
...
+- `true`, `false`, `null`, atau `undefined` (yang tidak ditampilkan)
+- Array *node* React lainnya
```
Comment (mhaidarhanif) [inline src/content/reference/react/isValidElement.md:116]: Need suggestion from @mazipan @resir014 @r17x @zainfathoni how to improve this sentence.

```suggestion
- Array *node* React lainnya
```

**Review by mhaidarhanif — COMMENTED** [review]
Need some improvements. This is a comment, no request for changes to avoid blocking.

File: src/content/reference/react/isValidElement.md [hunk 10]
```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
...
 
 ```js
 function MyComponent() {
-  return 42; // It's ok to return a number from component
+  return 42; // Tidak apa-apa untuk mengembalikan angka dari komponen
 }
 ```
 
-This is why you shouldn't use `isValidElement` as a way to check whether something can be rendered.
+Inilah sebabnya mengapa Anda tidak boleh menggunakan `isValidElement` sebagai cara untuk memeriksa apakah sesuatu dapat dirender.
```
Comment (mazipan) [inline src/content/reference/react/isValidElement.md:126]: Render pake italic

```suggestion
Inilah sebabnya mengapa Anda tidak boleh menggunakan `isValidElement` sebagai cara untuk memeriksa apakah sesuatu dapat di-*render*.
```

**Review by mazipan — APPROVED** [review]

File: src/content/reference/react/isValidElement.md (hunk 9 above)
Comment (mazipan) [reply src/content/reference/react/isValidElement.md:116]: This already good from me.

**Comment by mazipan** [comment]
Your call @resir014

**Review by mhaidarhanif — APPROVED** [review]

**Comment by mhaidarhanif** [comment]
@mazipan can merge without waiting for @resir014 if wanted to

**Comment by resir014** [comment]
We can merge now, I'll do it soon. @mhaidarhanif

**Pull request merged**